system/install.sh

```diff
@@ -0,0 +1,60 @@
+cd /
+echo Make sure you run this as root!
```

Comment:
Consider: https://unix.stackexchange.com/questions/28791/prompt-for-sudo-password-and-programmatically-elevate-privilege-in-bash-script
Or aborting if not root